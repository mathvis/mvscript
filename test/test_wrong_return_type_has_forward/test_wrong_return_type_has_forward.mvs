[fwd] func a() int;
func a() bool {
	return false;
}
