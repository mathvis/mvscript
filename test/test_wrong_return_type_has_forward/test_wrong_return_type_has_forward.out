error: Expected int as a return type but got bool
 --> test/test_wrong_return_type_has_forward/test_wrong_return_type_has_forward.mvs:2:1
  |
2 | func a() bool {
  | ^
  |
hint: Consider changing the return type, or checking the function signature.
